evil-collection.el

```diff
@@ -166,6 +166,24 @@ this confusing. It will be included if
   :type '(repeat (choice symbol sexp))
   :group 'evil-collection)
 
+(defcustom evil-collection-key-whitelist '()
+  "List of keys that may be used by evil-collection.
+This is a list of strings that are suitable for input to
+`kbd'. If there are no keys in the list, the whitelist will be ignored."
+  :type '(repeat string)
+  :group 'evil-collection)
+
+(defcustom evil-collection-key-blacklist '()
+  "List of keys that may not be used by evil-collection.
+This is a list of strings that are suitable for input to `kbd'."
+  :type '(repeat string)
+  :group 'evil-collection)
+
+(defvar evil-collection-bindings-record '()
```

Comment:
Name `evil-collection--bindings-record` because it should be private?

Reply:
ok